utils/frame/remote-externalities/src/lib.rs

```diff
@@ -101,9 +101,13 @@
 //! }
 //! ```
 
+// jsonrpsee_proc_macros generates faulty warnings: https://github.com/paritytech/jsonrpsee/issues/106
```

Comment:
Would be good if you provide follow ups for this.

Reply:
it's fixed on master but I haven't done another release yet :)